obspy/io/ascii/core.py

```diff
@@ -434,8 +443,9 @@ def _write_tspair(stream, filename, **kwargs):  # @UnusedVariable
             elif trace.data.dtype.name.startswith('float'):
                 dtype = 'FLOAT'
                 fmt = '%f'
```

Comment:
I'd change the default to `%+.10e` or at least to something that dynamically switches to scientific notation like `%.10g`.

Reply:
Okay!. Should I discriminate between float32('%+.8e') and float64('%+.10e') just like mseed2ascii?